raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -35,6 +35,7 @@ contract NettingChannelContract {
 
         data.token = Token(token_address);
         data.settle_timeout = timeout;
+        data.opened = block.number;
```

Comment:
I would really appreciate if this was renamed to `created`, since the channel is not really opened at this point (at least for the definition that a channel must have a balance to be considered open)